docs/DeveloperGuide.md

```diff
@@ -308,7 +315,7 @@ The following sequence diagram shows how the import operation works:
 #### 7.4.1 Implementation
 
 The proposed undo mechanism is facilitated by `ModelManager` and `History`.
-`History` combines `ReadOnlyMcGymmy`, `Predicate<Food>` and `MacroList` gotten from `ModelManager` together, then store multiple groups of different versions in a stack, with the most recent version on top.
+`History` combines `ReadOnlyMcGymmy`, `Predicate<Food>` and `MacroList` with `ModelManager`, then store multiple groups of different versions in a stack, with the most recent version on top.
```

Comment:
`History` contains a `Stack<Pair<McGymmy, Pair<Predicate<Food>, MacroList>>`, so this line is not correct

Reply:
It's fine if you discard this change here. I will update it myself in my #298 PR

Reply:
alright